generators/pattern-library/index.js

```diff
@@ -16,22 +16,23 @@ function init( config ) {
 
     config = configure( config );
 
-    if (config.error) {
+    if ( Array.isArray( config ) ) {
 
-        log.error(config.error);
-
-        return Promise.reject();
+        return new Promise( function ( resolve, reject ) {
+            return reject( config );
+        });
     }
 
     return globFiles( config )
+    .then( globFiles )
     .then( readSections )
     .then( parseSections )
     .then( render )
     .then( function ( html ) {
 
         log.info( 'Finished!' );
 
-        return output( html, config );
+        return html;
```

Comment:
If we take a look at output(), the function you've removed here, its returning the config file when format isn't declared. In our docs, we've also stated that we should be returning the expanded config object. So lets make sure we're returning the config object in this last return statement.